resources/splunk_app.rb

```diff
@@ -69,25 +65,43 @@ def parse_meta_access(access)
     end
   end
 
+  # Provider exclusive methods
+
   action_class do
     include CernerSplunk::ProviderHelpers
 
+    def upgrade_app(strategy)
+      require 'fileutils'
+
+      case strategy
+      when :prep
+        app_cache_path.mkpath
+        FileUtils.mv(app_path, app_cache_path + name)
+      when :keep_existing
+        FileUtils.cp_r(app_cache_path + name + 'local', app_path + 'local')
+        CernerSplunk::FileHelpers.deep_change_ownership(app_path + 'local', current_owner, current_group)
+
+        FileUtils.cp(app_cache_path + name + 'metadata/local.meta', app_path + 'metadata/local.meta')
```

Comment:
Shouldn't we do the same with [lookups](https://github.com/cerner/cerner_splunk/blob/stable/libraries/splunk_app.rb#L266-L268)?